docs/install/README.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Install
+description: Get up and running quickly by installing a Filecoin node to your system.
+---
+
+# Install Filecoin
+
+| Mine Filecoin                                                                                       | Build with Filecoin                                                                                                                 | For everyone else                                                                                                                                          |
+| --------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Are you setting up a miner to mine Filecoin? This section is for you!<br>[Find out more ↓](#miners) | Are you a developer looking to build an application on Filecoin? This section _might_ be for you.<br>[Find out more ↓](#developers) | Are you just looking to play around with Filecoin to see what all the fuss is about? This section is not for you.<br>[Find out more ↓](#for-everyone-else) |
+
+## Miners
+
+Miners host the infrastructure that holds the Filecoin network together! Without miners, there would only be a handful of very centralized, bootstrap servers being hosted by Protocol Labs. The first step to building a miner is [setting up a Lotus node ->](./lotus/)
+
+## Developers
+
+The entire Filecoin ecosystem is open to developers to build interesting and revolutionary applications with! However, depending on your use-case and experience, you may not need to host your own Filecoin node. If you've never built anything with Filecoin before, we recommend checking out the [Getting started](../build/) section to learn the ropes. In that guide, you'll use Powergate, a service that hosts Filecoin nodes for you, so you don't have to worry about spinning one up. Smaller-scale projects like weekend jams or hackathons may also benefit from using Powergate since there's very little set up time. If, however, you've done all this before and are building a fairly complex project, then you'll absolutely benefit from [setting up a Filecoin node ->](./lotus)
```

Comment:
```suggestion
The entire Filecoin ecosystem is open to developers to build interesting and revolutionary applications with! However, depending on your use-case and experience, you may not need to host your own Filecoin node. If you've never built anything with Filecoin before, we recommend checking out the [Getting started](../build/README.md) section to learn the ropes. In that guide, you'll use Powergate, a service that hosts Filecoin nodes for you, so you don't have to worry about spinning one up. Smaller-scale projects like weekend jams or hackathons may also benefit from using Powergate since there's very little set up time. If, however, you've done all this before and are building a fairly complex project, then you'll absolutely benefit from [setting up a Filecoin node ->](./lotus/README.md)
```